src/app/space/create/deployments/apps/deployment-card.component.ts

```diff
@@ -140,10 +145,22 @@ export class DeploymentCardComponent implements OnDestroy, OnInit {
     }
   }
 
+  openModal(): void {
+    this.modal.openModal(this);
+  }
+
+  lockAndDelete(): void {
+    this.collapsed = true;
+    this.deletingDeployment = true;
+  }
+
```

Comment:
If these functions aren't meant to be called from within the template (ie only from elsewhere within the component class), please make them private